### DISCUSSION TITLE
I just need Help!

### DISCUSSION THREAD

**Discussion by ifeobi** [community/community#45142] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Firstly, I am new to this. I am a lawyer transitioning into tech.

I built a static website using React Js (to get used to working with react). I deployed it on Pages (yeh!) and everything was working well until I put in a custom domain name with Namecheap and my website went blank (with only the <title /> displaying at the top).

I went into panic mode and watched a lot of youtube videos and totally destroyed everything. It's now worse. From blank to "There isn't a GitHub Pages site here."

Can someone help me here - the repo is https://github.com/ifeobi/IT-Solutions.git

**Comment by radar-blip** [comment]
Seems to be working fine now. 👍🏻

**Comment by Matary1** [comment]
its so ez my friend
